escrow/project/contracts/escrow-contract/src/interface.sw

```diff
@@ -2,14 +2,14 @@ library interface;
 
 dep data_structures;
 
-use data_structures::{Arbiter, Asset};
+use data_structures::{Arbiter, Asset, EscrowInfo};
 
 abi Escrow {
     /// Buyer accepts proposal to change arbiter details
     ///
     /// # Arguments
     ///
-    /// * `identifier` - Identifier used to find a specific escrow
+    /// * `identifier`: Identifier used to find a specific escrow
```

Comment:
The use of `:` differs from every other application in the repo, was this change done on purpose?

Reply:
> The use of `:` differs from every other application in the repo, was this change done on purpose?

Re: #421 , there are multiple other issues for this too.